tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -375,20 +418,21 @@ def test_fio_workload_simple(self, ripsaw, es, interface, io_pattern):
 
         """
 
+        # verify that there is an elasticsearch server for the benchmark
+        if not self.es:
```

Comment:
So one must have elasticsearch configured, either internal or external to the ocp cluster.   Is there an option to not have elasticsearch at all?   My understanding was that benchmark-operator did not require the use of elasticsearch...

Reply:
@lukebrowning , you are right, the benchmark-operator does not require using of elastic search server, but it make it more easy to retrieve test results from the ES server then parsing it from the test log.

for this reason we are require the use of the elastic search for the test.